Services.Test/Storage/StorageRecordTest.cs

```diff
@@ -44,7 +45,37 @@ public void ItConvertsFromDocumentDb()
         }
 
         [Fact]
-        public void ItConvertsFromDocumentDbRecord()
+        public void ItConvertsFromDocumentDbExpiredAndNotLocked()
+        {
+            // Arrange
+            var id = "123";
+            var data = "fooData";
+            var expirationUtcMsecs = Now;
+            var ownerId = "foo";
+            var ownerType = "bar";
+            var lockExpirationUtcMsecs = Now;
+
+            var document = new Document();
+            document.Id = id;
+            document.SetPropertyValue("Data", data);
+            document.SetPropertyValue("ExpirationUtcMsecs", expirationUtcMsecs);
+            document.SetPropertyValue("LockOwnerId", ownerId);
+            document.SetPropertyValue("LockOwnerType", ownerType);
+            document.SetPropertyValue("LockExpirationUtcMsecs", lockExpirationUtcMsecs);
+
+            // Act
+            var target = StorageRecord.FromDocumentDb(document);
+            Thread.Sleep(1);
```

Comment:
ditto

Reply:
I think that this Sleep makes sense. At the start of the test I'm setting the expiration to Now. The test later tests that the record has expired. If the test were to run sub-millisecond, it's possible that the record would not have expired and the Assert on line 42 would be hit.